### PR TITLE
update darts tests to match canonical.json

### PR DESCRIPTION
**Pull request by Tyresius92**
I noticed that various edge cases (x=4, y=4) were not covered in the tests for this exercise. Comparing to canonical.json showed that various situations were missing. I just added the missing tests.

### CODE REVIEW

**Comment by Tyresius92** [comment]
Definitely! Forgot to turn off JsPrettier.

**Comment by SleeplessByte** [comment]
Did you accidentally push dnd as well?

**Comment by Tyresius92** [comment]
Yes, that was an accident. Trying to roll it back now. Sorry about that!

**Comment by Tyresius92** [comment]
Okay, I think it's fixed now. Some wonky business happening with my local fork. I'll wait until this PR is (hopefully!) merged in, and then create a PR for the dnd one.

File: exercises/darts/darts.spec.js [hunk 1]
```diff
@@ -2,43 +2,78 @@ import { solve } from './darts';
...
+    const x = 4;
+    const y = 8;
+    const expected = 1;
+    expect(solve(x, y)).toEqual(expected);
+  });
+
+  xtest('Input is not a number', () => {
+    const x = 'WRONG';
+    const y = 10;
+    expect(solve(x, y)).toBeNull();
```
Comment (tejasbubane) [inline exercises/darts/darts.spec.js]: I suggest we remove this test case to make it in sync with canonical data.

Comment (SleeplessByte) [reply exercises/darts/darts.spec.js]: @Tyresius92 can you make this one last change? Ping me when done if I don't merge it in in timely fashion.

Comment (Tyresius92) [reply exercises/darts/darts.spec.js]: Yep! I should be able to take care of that this evening.
